.travis/metaSMT.sh

```diff
@@ -6,6 +6,7 @@ sudo apt-get -y install libboost1.55-dev libz3 libz3-dev libgmp-dev
 # Clone
 git clone git://github.com/hoangmle/metaSMT.git
 cd metaSMT
+git checkout 1f9e399
 git clone git://github.com/agra-uni-bremen/dependencies.git
```

Comment:
Are these dependencies stable?  I would also retrieve a certain version of this repository.